### PR TITLE
Code listing fix

### PR DESCRIPTION
**Pull request by marziel2001**

### CODE REVIEW

File: react-app/src/components/CodeListing.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { CSSProperties } from "react";
+import { parseCssEntries } from "../util/util";
+import clipboardSign from "../assets/clipboard-outline-svgrepo-com.svg";
+
+import styles from "../styles/flexbox/CodeListing.module.css";
+export function CodeListing({ cssProps }: { cssProps: CSSProperties }) {
+  function generateEntries() {
+    return parsedEntries.map((e, index) => (
```
Comment (RZEPI) [inline react-app/src/components/CodeListing.tsx]: in JS e is event shortcut, I suggest to use object desructurisation

File: react-app/src/components/UI/GridButton.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export default function GridConfigButton({
+  name,
+  handleClick,
+  style,
+}: {
+  name?: string;
```
Comment (RZEPI) [inline react-app/src/components/UI/GridButton.tsx]: why the name is optional here? Every button should have some caption

File: react-app/src/utils.ts
Comment (RZEPI) [inline react-app/src/utils.ts:1]: no use in empty files

**Review by RZEPI — CHANGES_REQUESTED** [review]

File: react-app/src/components/UI/GridButton.tsx (hunk 2 above)
Comment (marziel2001) [reply react-app/src/components/UI/GridButton.tsx]: who knows...

File: react-app/src/utils.ts
Comment (marziel2001) [reply react-app/src/utils.ts:1]: of corse sir

File: react-app/src/components/CodeListing.tsx (hunk 1 above)
Comment (marziel2001) [reply react-app/src/components/CodeListing.tsx]: Name fixed to "entry", but I won't change map to destruicturisation. I thing its better there

**Review by RZEPI — APPROVED** [review]

**Review by NOTSAHLO — APPROVED** [review]

**Review by grczk — APPROVED** [review]

**Pull request merged**